### PR TITLE
Release/wanderer

### PR DESCRIPTION
**Pull request by jtbaird**
# PR Details
## Description
This PR is to merge the Wanderer release (carma-system-3.4.0) into the master branch.

## Related Issue
None

## Motivation and Context
Purpose of the Wanderer release is to incorporate four major enhancements into the CARMA system. First, a steering limiter was added in order to prevent Carma from taking turns that present a risk of vehicle rollover. Second, updates were made to the health_monitor to allow for graceful handling of sensor failure, including control handover to the human driver. Third, carma-messenger was updated to detect nearby automated vehicles via DSRC. And fourth, functionality was added to carma-messenger to allow for requesting, receiving, and displaying truck safety info from a passing Carma truck via DSRC while vehicles are in motion. Along with the above enhancements, several bug fixes and CI related enhancements are included in this release.

## How Has This Been Tested?
Per the release test plan.

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# Docker Compose Spec Version
+version: '2'
+
+services:
+  roscore:
+    image: usdotfhwastol/carma-base:CARMASystem_3.3.0
```
Comment (maefromm) [inline carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml:20]: Should this be updated to 3.4.0?

File: carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=192.168.0.51
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /opt/carma/.ros:/opt/carma/.ros
+    command: wait-for-it.sh localhost:11311 -- roslaunch /opt/carma/vehicle/config/carma-messenger-docker.launch
+
+  cohda_dsrc_driver:
+    image: usdotfhwastol/carma-cohda-dsrc-driver:CARMASystem_3.3.0
```
Comment (maefromm) [inline carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml:46]: same question here

**Review by maefromm — CHANGES_REQUESTED** [review]

File: carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml (hunk 2 above)
Comment (jtbaird) [reply carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml:46]: good catch. my bad the messenger layout is just way different than the other repos

**Comment by maefromm** [comment]
image
Does this need to be updated as well?

**Comment by jtbaird** [comment]
> image
> Does this need to be updated as well?

good catch. fixed and pushed.

**Review by maefromm — APPROVED** [review]

**Pull request merged**